library/std/src/sys/fd/unix.rs

```diff
@@ -87,6 +87,19 @@ const fn max_iov() -> usize {
     16 // The minimum value required by POSIX.
 }
 
+#[cfg(not(any(
+    all(target_os = "linux", not(target_env = "musl")),
+    target_os = "android",
+    target_os = "hurd"
+)))]
+use libc::pread as pread64;
+#[cfg(any(
+    all(target_os = "linux", not(target_env = "musl")),
+    target_os = "android",
+    target_os = "hurd"
+))]
+use libc::pread64;
```

Comment:
Since this is getting moved anyway, mind simplifying this with `cfg_select!`?

Reply:
Sure. Did I pick the right place for `#![feature(cfg_select)]`?